test/functional/apps/dashboard_elements/controls/options_list/options_list_suggestions.ts

```diff
@@ -119,6 +119,27 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await dashboardControls.optionsListPopoverSetSort({ by: '_count', direction: 'desc' });
         await testSubjects.missingOrFail('dashboardUnsavedChangesBadge');
       });
+
+      it('can sort numeric options lists suggestions', async () => {
```

Comment:
Functional tests are more expensive (meaning time) to maintain and run. I think this case would be better covered by adding a numeric test case to src/plugins/controls/server/options_list/suggestion_queries/options_list_search_suggestions.test.ts.

Reply:
@ThomThomson suggested having at least one smoke test to make sure that the options list popover can be opened without errors, and that is why I added this sorting functional test - I am personally in favour of keeping at least one of these two new tests in for that reason.